src/components/user/LoginForm.js

```diff
@@ -2,13 +2,15 @@ import React from 'react';
 import { func, string, bool } from 'prop-types';
 import { Field, reduxForm } from 'redux-form/immutable';
 import { Link } from 'react-router-dom';
+import FacebookLogin from 'react-facebook-login';
 import {
   injectIntl,
   intlShape,
   defineMessages,
   FormattedMessage
 } from 'react-intl';
 
+import fbID from '../../constants/constants';
```

Comment:
Names should match with the ones in the file.

Also when you're exporting more tha one thing you'll need the `{ ... }` notation to pull things out of it.